homework/App.js

```diff
@@ -18,17 +18,42 @@ class App {
 
     const root = document.getElementById('root');
 
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
+    const container = Util.createAndAppend('div', root, {
+      class: 'container',
+    });
+
+    const header = Util.createAndAppend('header', container, {
+      class: 'header',
+    });
+
+    Util.createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = Util.createAndAppend('select', header, {
+      id: 'list',
+    });
+    select.addEventListener('change', () => {
+      this.fetchContributorsAndRender(select.value);
+    });
+
+    Util.createAndAppend('div', container, { id: 'container-repo' });
 
     try {
```

Comment:
You can perhaps think of extracting this logic out in a function as well. Similar to how you have `fetchContributorsAndRender`.